bool record_time    # True to record actual time
---
bool resp
